test/data/repositories/rhel-9.json

```diff
@@ -0,0 +1,66 @@
+{
+    "aarch64": [
+        {
+            "name": "baseos",
+            "baseurl": "http://download-node-02.eng.bos.redhat.com/rhel-9/composes/RHEL-9/RHEL-9.0.0-20210228.7/compose/BaseOS/aarch64/os",
```

Comment:
This is fine for now I think, but we should ask @dvdhrm to add this to his snapshots so we can use a RHEL9 snapshot here as we do for RHEL8.